src/plugins/wb-data-scrub/wb-data-scrub-en.hbs

```diff
@@ -0,0 +1,85 @@
+			<input class="form-control" id="email" name="email" type="email" data-rule-wbscrub />
+		</div>
+		<div class="form-group">
+			<label for="search"><span class="field-name">Search</span></label>
+			<input class="form-control" id="search" name="search" type="search" data-rule-wbscrub />
+		</div>
+		<div class="form-group">
+			<label for="comments"><span class="field-name">Comments</span></label>
+			<textarea class="form-control" id="comments" name="comments" data-rule-wbscrub rows="4" cols="50" ></textarea>
+		</div>
+		<div class="form-group">
+			<label for="noScrubField"><span class="field-name">No scrub field example</span></label>
+			<input class="form-control" id="noScrubField" name="noScrubField" type="text"/>
+		</div>
+		<button type="submit" class="btn btn-primary">Submit</button>
+	</div>
+</form>
+
+
+<details class="mrgn-tp-md mrgn-bttm-md">
+	<summary>View code</summary>
+	<pre><code>&lt;form action="wb-data-scrub-en.html" &gt;
+	&lt;div class="form-content"&gt;
+		&lt;div class="form-group"&gt;
+			&lt;label for="preferredNumber"&gt;Preferred: &lt;span class="field-name"&gt;Phone Number&lt;/span&gt;&lt;/label&gt;
+			&lt;input class="form-control" id="preferredNumber" name="preferredNumber" type="tel" data-rule-wbscrub /&gt;
+		&lt;/div&gt;
+		&lt;div class="form-group"&gt;
+			&lt;label for="email"&gt;&lt;span class="field-name"&gt;Email Address&lt;/span&gt; (test@domaine)&lt;/label&gt;
+			&lt;input class="form-control" id="email" name="email" type="email" data-rule-wbscrub /&gt;
+		&lt;/div&gt;
+		&lt;div class="form-group"&gt;
+			&lt;label for="search"&gt;&lt;span class="field-name"&gt;Search&lt;/span&gt;&lt;/label&gt;
+			&lt;input class="form-control" id="search" name="search" type="search" data-rule-wbscrub /&gt;
+		&lt;/div&gt;
+		&lt;div class="form-group"&gt;
+			&lt;label for="comments"&gt;&lt;span class="field-name"&gt;Comments&lt;/span&gt;&lt;/label&gt;
+			&lt;textarea class="form-control" id="comments" name="comments" data-rule-wbscrub rows="4" cols="50"&gt;&lt;/textarea&gt;
+		&lt;/div&gt;
+		&lt;div class="form-group"&gt;
+			&lt;label for="noScrubField"&gt;&lt;span class="field-name"&gt;No scrub field example&lt;/span&gt;&lt;/label&gt;
+			&lt;input class="form-control" id="noScrubField" name="noScrubField" type="text"/&gt;
+		&lt;/div&gt;
+		&lt;button type="submit" class="btn btn-primary"&gt;Submit&lt;/button&gt;
+	&lt;/div&gt;
+&lt;/form&gt;</code></pre>
+</details>
+
+<h2>Test case</h2>
+<p>Fill the following fields with the following values:</p>
+<ul>
+	<li>Phone: <code>514 111 1111</code></li>
+	<li>Email: <code>1@example.com</code></li>
+	<li>Search: <code>Passport Nr: AB123456</code></li>
+	<li>Comments: <code>visa: 4916 0739.3667-6891, master:5428735149026050, SIN:123 123-123, ZIP:K2C 3N2, Driving Licence Nr:P12345678912345, Bank account example: 003-1234567</code></li>
+	<li>No scrub field example: <code>John Doe</code></li>
+</ul>
```

Comment:
@duboisp We have some test cases strings here. Would you like @polmih to move them somewhere else instead?

Reply:
Yes, I will provide some instruction in the other PR.